YvesIraguha/readit_react_front_end
# # 165466659 Setup Project Structure

#### What does this PR do?
- Create a project structure
- Set up react-redux 
#### Description of Task to be completed?
- Create a project structure
- Set up react-redux 
- Configure webpack 
- Test web pack and initial configurations
#### How should this be manually tested?
- Clone the repo and check out `ch-setup-project-structure-165466659`
- Install all dependencies by running `yarn `
- Start the project, 
- Navigate to a different route and click on the button. 
- Open dev tools you shall see the state is updated as you click. 

#### What are the relevant pivotal tracker stories?
#165466659
